design/high-level-reactivity.md

```diff
@@ -205,10 +248,19 @@ context.invalidateDependency(notificationDepKeyForUser(userId));
 
 Hopefully, with time, we can make more and more invalidations automatic, but it’s always good to have an “escape hatch” for more complex situations where the developer needs all the control they can get.
 
+// Add a section here: This is why we said, you need to add custom keys. 
+// Then we can have some examples like this:
+
+1. Stop reactivity. Just publish a depKey which is random. Do not invalidate that key.
```

Comment:
Yeah I was thinking in this case the dep key can just be a sentinel value, like `null` or something.